rdr_service/genomic/genomic_job_controller.py

```diff
@@ -297,41 +304,116 @@ def ingest_member_ids_from_aw1_raw_table(self, member_ids):
 
                 for r in update_recs:
 
-                    # Set RDR attributes: aw1_file_processed_id, job_run_id, gc_site
-                    self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
+                    # AW1
+                    if n == 1:
+                        self.set_rdr_aw1_attributes_from_raw(r, file_proc_map)
 
-                    # Set attributes
-                    member = self.set_aw1_attributes_from_raw(r)
+                        self.set_aw1_attributes_from_raw(r)
 
-                    s.merge(member)
-                    completed.append(member.id)
+                    # AW2
+                    else:
+                        self.preprocess_aw2_attributes_from_raw(r, file_proc_map)
 
-        # Output results
-        result_msg = ''
-        result_msg += 'Ingestion From Raw Results:'
-        result_msg += f'    Updated IDs: {completed}'
-        result_msg += f'    Missing IDs: {missing}'
-        result_msg += f'    Multipls found for IDs: {multiples}'
+                        metrics_obj = self.set_validation_metrics_from_raw(r)
 
-        return result_msg
+                        metrics_obj = s.merge(metrics_obj)
+                        s.commit()
+                        metrics.append(metrics_obj.id)
+
+                    s.merge(r[0])
```

Comment:
General comment for future consideration:  moving forward, I think it helps make code easier to digest/understand on first read  if variable names (even the with/as aliases) are more descriptive (vs. single-char names)

Reply:
Yes, that is a good practice. I'll update "s" and "r" to be "session" and "record", respectively. The variable "n" represents the "n" in AWN and is defined in the docstring. I'll update the comment to make that more clear.